src/Compatibility/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -138,6 +138,9 @@
   <data name="MemberExistsOnLeft" xml:space="preserve">
     <value>Member '{0}' exists on the left but not on the right</value>
   </data>
+  <data name="MemberExistsOnRight" xml:space="preserve">
+    <value>Member '{0}' exists on the right but not on the left</value>
```

Comment:
Unreleated to this PR, but are we planning on parameterizing the names for right/left so that the diagnostics are more readable?

Reply:
Yes. I'm working on that. Should have a PR out soon. That's on the timeline for Preview 6.